bukkaApp/Mybukka-Frontend
# #163359748: Build Out Homepage

#### What does this PR do?
- Build out HomePage

#### Description of the task to be completed?
- build out reusable components
- build out homepage
- write unit tests

#### How can this manually be tested?
- clone this repo
```
$ git clone https://github.com/bukkaApp/Mybukka-Frontend
```
- install all dependencies
```
$ npm i
```
- start Application
```
$ npm start:dev
```
- log on to _localhost:PORT_ to see page created

#### Any background context you want to provide?

#### Relevant PT story(ies)
#163359748

#### Screenshots (if appropriate)
![screencapture-0-0-0-0-7700-2019-01-30-04_24_57](https://user-images.githubusercontent.com/24938186/51956331-88eca980-2447-11e9-8e27-a498dc6fe8f6.png)


#### Questions
